### DISCUSSION TITLE
How to answer Width of segregated cycleways?

### DISCUSSION THREAD

**Discussion by timothywashere** [streetcomplete/StreetComplete#3910] [General]
re #698

When the width question is asked for cycleways that are segregated I am unsure whether to measure the whole path or only the cycle section?

Also when a way is tagged only as highway=cycleway and there is no tag to say whether pedestrians are allowed or not, maybe SC should ask? I only found #3785 about sidewalks on cyclepath but some segregated are just painted on and some cyclepaths aren't segregated and could just be amended with a simple question?

**Comment by westnordost** [comment]
Only the cycleway section
